internal/runners/export/config.go

```diff
@@ -0,0 +1,66 @@
+package export
+
+import (
+	"encoding/json"
+	"strings"
+
+	"github.com/ActiveState/cli/internal/captain"
+	"github.com/ActiveState/cli/internal/config"
+	"github.com/ActiveState/cli/internal/locale"
+	"github.com/ActiveState/cli/internal/output"
+)
+
+const (
+	FilterDir = "dir"
+)
+
+func RecognizedFilters() string {
+	filterLookup := []string{
+		FilterDir,
+	}
+	return strings.Join(filterLookup, ",")
+}
+
+type Config struct {
+	out output.Outputer
+}
+
+type ConfigParams struct {
+	Filter string
+}
+
+func NewConfig(prime primeable) *Config {
+	return &Config{prime.Output()}
+}
+
+func (c *Config) Run(cmd *captain.Command, params ConfigParams) error {
+	if params.Filter == FilterDir {
+		c.out.Print(config.ConfigPath())
```

Comment:
Rather than switching based on filter we should actually be filtering. Ie. the standard output is a struct with the "dir" key, and if filter=dir is given we actually filter to that level.